src/Files.App/Views/ModernShellPage.xaml.cs

```diff
@@ -1089,6 +1089,9 @@ public void NavigateToPath(string? navigationPath, Type? sourcePageType, Navigat
 					return;
 				}
 
+				if (string.IsNullOrEmpty(navigationPath))
```

Comment:
Isn't this already checked on line 1077?

Reply:
It is not, it's actually part of a bigger condition : 
`(string.IsNullOrEmpty(navigationPath) || string.IsNullOrEmpty(FilesystemViewModel?.WorkingDirectory) || navigationPath.TrimEnd(Path.DirectorySeparatorChar).Equals(FilesystemViewModel.WorkingDirectory.TrimEnd(Path.DirectorySeparatorChar),StringComparison.OrdinalIgnoreCase))`